unity/Holo/Assets/VTKConverter/Kitware.VTK.dll.meta

```diff
@@ -0,0 +1,47 @@
+fileFormatVersion: 2
```

Comment:
Do we need all these files from Kitware in Unity project since we've pushed the application outside of Unity project?

Reply:
I wanted to contain all the files needed to preprocess in a single Unity project so I've put the VTKConverter with DLLs necessary to run it in this folder. Unity ignores the DLLs, but still creates metas as it acknowledges them as part of the project. In the next PR this folder is moved to a separate Unity project so it will be gone from 'holo'.